components/Greeting/Greeting.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from "../Greeting/Greeting.module.css";
+export type Name = {
```

Comment:
`Name` is misleading because you describe an object with a property called "name" here.
I would give it a different name.

Imagine adding a second property to your component.